Kirill_Lazniuk/Homework-12/HW-12.py

```diff
@@ -0,0 +1,17 @@
+import random
+import sys
+
+try:
+    min_limit = int(input("Введіть мінімальну межу: "))
+    max_limit = int(input("Введіть максимальну межу: "))
+    
+    if min_limit >= max_limit:
+        print("Помилка: Мінімальна межа повинна бути меншою за максимальну.")
+        sys.exit()
```

Comment:
up to you: put "1" as to indicate that this is an error
```suggestion
        sys.exit(1)
```